framework/widgets/LinkPager.php

```diff
@@ -74,6 +75,18 @@ class LinkPager extends Widget
      * @var string the CSS class for the disabled page buttons.
      */
     public $disabledPageCssClass = 'disabled';
+    
+    /**
+     * @var array An array with options to pass to the disabled tag which is generated inside the disabled list element.
+     * In order to to customize the tag name itself use the array key `tag` and value for its tag name value.
+     * 
+     * ```php
+     * $disabledPageElementOptions = ['tag' => 'div', 'class' => 'disabled-div'];
+     * ```
+     * @since 2.0.11
+     */
+    public $disabledPageElementOptions = [];
```

Comment:
It was suggested by @SilverFire. But its also hard to imagine what the other properties stand for.